server/services/stats/side/test.go

```diff
@@ -65,6 +72,30 @@ func (t *testNrClient) GetServersStats() ([]newrelic.Server, error) {
 	return servers, nil
 }
 
+type testTcClient struct {
+	CountGetTask int8
+}
+
+func NewTestTcClient() TcClient {
+	return &testTcClient{
+		CountGetTask: 5,
+	}
+}
+
+func (t *testTcClient) GetBuildStats() ([]*teamcity.Build, error) {
+	tasks := make([]*teamcity.Build, t.CountGetTask, t.CountGetTask)
```

Comment:
https://play.golang.org/p/Pp1L8L2cKg

you can skip capacity argument if it's equal to length